### ISSUE TITLE
[yugabyted] Refreshing the UI causes failure

### ISSUE THREAD

**Issue by rkarthik007** [yugabyte/yugabyte-db#3206]
The steps to repro are:
* Install the DB using `yugabyted`
* Create the [northwind demo](https://docs.yugabyte.com/latest/sample-data/northwind)
* Issue the DDL create command as per [this step](https://docs.yugabyte.com/latest/sample-data/northwind/#4-build-the-tables-and-objects): `\i share/northwind_ddl.sql`
*  Refresh the UI when the tables are being created

This causes the cluster to fail. Logs are attached.
[logs.tar.gz](https://github.com/yugabyte/yugabyte-db/files/3990514/logs.tar.gz)

**Comment by iSignal** [comment]
```
[yugabyted:1685] 2019-12-20 22:14:21,346 INFO:  | 745.1s | Checking play_evolutions table
[yugabyted:1685] 2019-12-20 22:14:23,093 INFO:  | 746.9s | Finished play evolutions table
[yugabyted:1685] 2019-12-20 22:14:23,093 INFO:  | 746.9s | Setting up admin console schema...
[yugabyted:1685] 2019-12-20 22:14:23,094 ERROR:  | 746.9s | Traceback (most recent call last):
  File "./bin/yugabyted", line 952, in run
    args.func()
  File "./bin/yugabyted", line 140, in start
    self.start_processes()
  File "./bin/yugabyted", line 458, in start_processes
    Output.begin_replace_line("Installing database...")
  File "./bin/yugabyted", line 1659, in begin_replace_line
    sys.stdout.flush()
IOError: [Errno 9] Bad file descriptor
```

This seems to be a recurring issue that many have run into. The pattern is that doing some DDL or other intensive operations on the db causes yugabyted to return a failure from its periodic ysqlsh list play tables call. The surprising part is that this then translates to a bad file descriptor error for writing to sys.stdout, which doesn't make any sense.

Removing the periodic play table list would fix this but it would be good to also understand how python got into a state of "Bad file descriptor" for sys.stdout

**Comment by WesleyW** [comment]
Fixed as part of 07f07aa, was a problem with YSQL/DocDB mismatch
